packages/frameworks/frameworks.json

```diff
@@ -27,6 +27,34 @@
       }
     }
   },
+  {
+    "name": "Blitz.js",
```

Comment:
I believe that Blitz.js needs to be _before_ Next.js, otherwise a user that has both dependency will get Next.js instead of Blitz.js

Reply:
Fixed